Tools/gitlab.md

```diff
@@ -0,0 +1,52 @@
+# GitLab
+
+Cohorts can use GitLab as an alternative repository host instead of GitHub.
+GitLab is open source and can be self hosted, but we use the main instance at <https://gitlab.com/>.
+
+## Alternative setup for Module 1
+
+You can use GitLab instead of GitHub for your project exercise.
+It is largely a drop in replacement, but where the project exercise instructions are GitHub specific we provide alternative instructions here.
+
+### Setup: Checkout the starter project
+
+[Sign up for a GitLab account here](https://gitlab.com/users/sign_up).
+
+Follow the Module 1 Project exercise instructions up to and including cloning the starter project.
+
+```sh
+git clone https://github.com/CorndelWithSoftwire/DevOps-Course-Starter.git
+cd DevOps-Course-Starter
+```
+
+Now, create a new repository on GitLab.com:
+
+* Go to the [Create a blank project](https://gitlab.com/projects/new#blank_project) page on GitLab
+* Enter a name for your project, e.g. "DevOps Project Exercise"
+* Set the visibility to "Public", and untick the other options, then click "Create project"
+
+![GitLab create a blank project screen](../assets/images/gitlab_new_project.png)
+
+Follow the "Command line instructions" on the GitLab page for your new project to push your existing repository, the commands should look something like:
+
+```sh
+git remote remove origin # Unlink the original GitHub repository
+git remote add origin https://gitlab.com/your-username/devops-project-exercise.git # Replace "your-username" as appropriate, and make sure the repository name is correct.
+git push --set-upstream origin master # Push the current branch
+```

```

Comment:
The new project splash page instructions are a bit more complicated than GitHub's, and don't really look much like this, but it should be enough for them to work it out.

Reply:
Then should you say to ignore GitLab's instructions?

Reply:
Yeah, probably, this is what the page looks like 
![image](https://user-images.githubusercontent.com/35066342/142631894-f35767a1-ddcf-442e-bfc5-45ee8f163d67.png)